### PR TITLE
Flattened JSON output and stringify early.

### PR DESCRIPTION
**Pull request by mollyling**
Makes it easier to parse from R and probably other stat tools.

[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-networking/206)

### CODE REVIEW

File: src/rtc-to-net/rtc-to-net.ts [hunk 1]
```diff
@@ -23,28 +23,18 @@ module RtcToNet {
...
-    received :number;
-    buffered :number;
-    queue :HandlerQueueSnapshot;
-  }
-
   export interface SessionSnapshot {
     name :string;
-    channel :DataChannelSnapshot;
-    socket :SocketSnapshot;
+    timestamp: number;
```
Comment (trevj) [inline src/rtc-to-net/rtc-to-net.ts:32]: Short comment here.

This is when the log entry was generated vs. the timestamp included with the log line?

File: src/rtc-to-net/rtc-to-net.ts [hunk 2]
```diff
@@ -175,7 +165,7 @@ module RtcToNet {
       });
       var writeSnapshot = () => {
         this.getSnapshot().then((snapshot:RtcToNetSnapshot) => {
-          log.info('snapshot: %1', [JSON.stringify(snapshot)]);
+          log.info('snapshot: ' + JSON.stringify(snapshot));
```
Comment (trevj) [inline src/rtc-to-net/rtc-to-net.ts]: Curious why you changed this -- the `%X` placeholders are intended to be faster, by avoiding the string concatenation.

**Comment by trevj** [comment]
A couple of tiny things, otherwise :+1: and happy to see this evolving!

File: src/rtc-to-net/rtc-to-net.ts (hunk 2 above)
Comment (mollyling) [inline src/rtc-to-net/rtc-to-net.ts]: Two reasons:
- First, logging ain't working right, so I want to simplify what we're doing with logging.  Hopefully it'll eventually help
- Let's just construct and keep the string, and not retain the entire JS object.

File: src/rtc-to-net/rtc-to-net.ts (hunk 1 above)
Comment (mollyling) [inline src/rtc-to-net/rtc-to-net.ts:32]: Comment added.

File: src/rtc-to-net/rtc-to-net.ts (hunk 2 above)
Comment (trevj) [inline src/rtc-to-net/rtc-to-net.ts]: I don't see how it'll eventually help. FYI, @jpevarnek made some recent improvements and now you could also just do:

``` typescript
 log.info('snapshot: %1', snapshot);
```

Or even just:

``` typescript
 log.info(snapshot);
```

**Pull request merged**